docs/tutorial/basic-dl-query.md

```diff
@@ -0,0 +1,46 @@
+# DL query
+
+This tutorial is based off https://ontology101tutorial.readthedocs.io/en/latest/DL_QueryTab.html
+Created by: Melissa Haendel, Chris Mungall, David Osumi-Sutherland, Matt Yoder, Carlo Torniai, and Simon Jupp
+
+## DL query tab
+
+The DL query tab shown below provides an interface for querying and searching an ontology. The ontology must be classified by a reasoner before it can be queried in the DL query tab.
+
+For this tutorial, we will be using cc.owl which can be found [here](https://github.com/OHSUBD2K/BDK14-Ontologies-101/tree/master/BDK14_exercises/basic-dl-query).
+
+Go to the “basic-dl-query” folder and open “cc.owl”. Run the reasoner. Navigate to the DL Query tab.
+
+![](../images/tutorials/dl-query/Figure57.png)
+
+Type “organelle” into the box, and make sure “subclasses” and “direct subclasses” are ticked.
+
+![](../images/tutorials/dl-query/Figure58.png)
+
+You can type any valid OWL class expression into the DL query tab. For example, to find all classes whose members are part_of a membrane, type “part_of some membrane” and click ‘execute’. Note the linking underscore for this relation in this ontology. Some ontologies do not use underscores for relations, whereby you’d need single quotes (i.e. ‘part of’).
+
+![](../images/tutorials/dl-query/Figure59.png)
+
+The OWL keyword “and” can be used to make a class expression that is the intersection of two class expressions. For example, to find the classes in the red area below, we want to find subclasses of the intersection of the class ‘organelle’ and the class ‘endoplasmic reticulum part’
```

Comment:
Your quotes look suspicious, like from a different character encoding. Shouldnt they look more like " (more 'straight')?

Reply:
will go through and change it all